linkml/generators/graphqlgen.py

```diff
@@ -50,13 +53,38 @@ def visit_class_slot(self, cls: ClassDefinition, aliased_slot_name: str, slot: S
             slotrange = slotrange + "!"
         return f"\n    {lcamelcase(aliased_slot_name)}: {slotrange}"
 
+    def visit_enum(self, enum: EnumDefinition):
+        if not enum.permissible_values:
+            if self.fail_on_missing_permissible_values:
+                raise NotImplementedError(
+                    "Enumerations using `reachable_from` instead of `permissible_values` to "
+                    + "specify permissible values is not supported yet."
+                )
+            else:
+                print(
+                    "Warning!! Enumerations using `reachable_from` instead of `permissible_values` "
+                    + f"are not supported yet. Enumeration '{enum.name}' will be simply ignored.",
+                    file=sys.stderr,
+                )
+                return ""
+        values = "\n    ".join([v.upper() for v in enum.permissible_values])
+        out = f"enum {camelcase(enum.name).replace(' ','')}\n  {{\n    {values}\n  }}\n\n"
+        return out
+
 
 @shared_arguments(GraphqlGenerator)
 @click.command(name="graphql")
+@click.option(
+    "--fail-on-missing-permissible-values/--no-fail-on-missing-permissible-values",
```

Comment:
Wasnt there another problem with empty interfaces and etc? We might want to name this "fail empty elements" to account for the general pattern if graphql not allowing any schema element to be empty

Reply:
Not applicable anymore. CLI parameters reduced to `--strict-naming`.